environment-dev.yml

```
@@ -25,9 +25,11 @@ dependencies:
   - requests
   - timezonefinder
   #
-  # test
+  # test/dev
   - pytest
   - pytest-xdist
+  - pytest-mock
```

We try for alphabetical order in sections
```suggestion
  - pytest-mock
  - pytest-xdist
```

---

Considering the mock testing is not needed. I removed this dependency. Resolving